compiler/noirc_frontend/src/hir/type_check/expr.rs

```diff
@@ -826,7 +826,8 @@ impl<'interner> TypeChecker<'interner> {
     ) -> Option<HirMethodReference> {
         match object_type {
             Type::Struct(typ, _args) => {
-                match self.interner.lookup_method(typ.borrow().id, method_name) {
+                let id = typ.borrow().id;
```

Comment:
Did you do this for formatting?

Reply:
Yes, with the new `object_type` and `false` parameters it would have been split onto multiple lines otherwise.